portability-transfer/src/main/java/org/datatransferproject/transfer/JobProcessor.java

```diff
@@ -124,6 +142,7 @@ void processJob() {
           JobMetadata.getExportService(),
           JobMetadata.getImportService());
       JobMetadata.getStopWatch().start();
+      sendSignals(jobId, exportAuthData, importAuthData, JobLifeCycle.JOB_STARTED(), monitor);
       copier.copy(exportAuthData, importAuthData, jobId, exportInfo);
```

Comment:
One critical change: can you please add flag guarding here. This is that settings/value propagating mechanism I shared. Just wrap this new line and the next two new lines in a conditional ([eg](https://github.com/dtinit/data-transfer-project/blob/b92d3317efdbe127f101fab9cfb693e0d94b9bb7/extensions/data-transfer/portability-data-transfer-google/src/main/java/org/datatransferproject/datatransfer/google/GoogleTransferExtension.java#L120) `if (context.getSetting("useJobSignal", false)) {`)